cou/cli.py

```diff
@@ -197,6 +215,12 @@ async def entrypoint() -> None:
         logger.error("Error occurred in Juju's Python library.")
         logger.error(exc)
         sys.exit(1)
+    except KeyboardInterrupt as exc:
+        # NOTE(rgildein): if spinner_id is not None it means that indicator was not finished
+        if progress_indicator.spinner_id is not None:
+            progress_indicator.fail()
+        print(str(exc) or "charmed-openstack-upgrader has been terminated")
+        sys.exit(130)
```

Comment:
`130` is fore `SIGINT` and `143` for `SIGTERM`. Would be nice if we can reflect that like

```suggestion
        sys.exit(128 + <SIG_SIGNAL>)
```

Reply:
That is good idea, I will take a look at it.